### ISSUE TITLE
accessibility: GestureRecognizer and TalkBack onTapHint

### ISSUE THREAD

**Issue by khsbory** [flutter/flutter#115786]
Basically TalkBack speak hint, double tap to activate, once I focus clickable widget.
In other words, if focused object is clickable or long clickable, TalkBack speak related hint double tap to activate or double tap and hold to long press.
In flutter, once I use native button with onPress property to handle button action, TalkBack speak hint properly such as double tap to activate.
Also I can change hint string with onTapHint with Semantics.
However if I should use custom button with GestureRecognizer with onTap, TalkBack does not recognize hint.
Also even if I add onTapHint with Semantics widget, there is no work.
Of course GestureRecognizer is touch event and it has no clickable, it is right that TalkBack does not speak hint.
But I think onTapHint should work with GestureRecognizer to provide additional hint information.
As a reference, hint property with Semantics should be fixed, too.
Because if I apply this hint string, TalkBack read it with label.
In other words, even if I turn off speak hint with TalkBack settings, the hint string is spoken.
Lastly, once I should use GestureRecognizer with onTap, if there is a way let TalkBack speak hint message, please let me know.
Thank you.

**Comment by huycozy** [comment]
Hi @khsbory 
Please update the issue by providing the information in the format below so we can verify the issue properly:
- A [completed and minimal reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example)
- The output of `flutter doctor -v`
- Reproduction steps, Expected results, Actual results

Thank you!
